package.json

```diff
@@ -32,7 +32,7 @@
     "@giphy/react-components": "5.6.0",
     "@giphy/react-native-sdk": "1.7.0",
     "@hapi/bourne": "2.0.0",
-    "@jitsi/excalidraw": "https://github.com/jitsi/excalidraw/releases/download/v0.0.11/jitsi-excalidraw-0.0.11.tgz",
+    "@jitsi/excalidraw": "https://github.com/jitsi/excalidraw/releases/download/v0.0.12/jitsi-excalidraw-0.0.12.tgz",
```

Comment:
Why is this bump necessary? Is it related to this fix? If not, please make a separate PR mentioning what it adds / fixes.

Reply:
I thought of it as a singular approach to improve debugging on this side, it can be moved in a separate PR however